### DISCUSSION TITLE
Just starting

### DISCUSSION THREAD

**Discussion by MartinsAloysius** [Cyfrin/foundry-full-course-cu#2677] [General]
Hi everyone after a whole jumping from web dev i've decided to stick to this. Let's all have fun.

**Comment by BowtiedHarpyEagle** [comment]
Hi Martin. Best of luck with your studies.

**Comment by MartinsAloysius** [comment]
Thank you 😊🙏

**Comment by EngrPips** [comment]
Best of luck, Martin.

**Comment by MartinsAloysius** [comment]
Thanks 😊

**Reply by EngrPips** [reply]
Anytime buddy.

**Comment by MartinsAloysius** [comment]
Awesome, thanks!

**Comment by MattSWeb3** [comment]
@MartinsAloysius Best of luck man. I'm new to all of this too. Loving the course so far.

**Reply by MartinsAloysius** [reply]
Lovely

**Comment by MartinsAloysius** [comment]
Let's be buddies 😜

**Comment by Ibnetgm** [comment]
Hi buddies im Ibrahim musa from Nigeria i am here to learn more about smart contract development please guys i need your support

**Reply by EngrPips** [reply]
Welcome aboard, Ibrahim. We wish you the best.

**Reply by MartinsAloysius** [reply]
Hey Ibrahim

**Comment by Qunleez** [comment]
Hi buddies I'm Quadri from Nigeria am a newbie in web 3 pls i need someone that will be putting me through

**Reply by MartinsAloysius** [reply]
Hey man good to have you around, I'm Nigerian too

**Comment by finestxxx** [comment]
hello everyone, i am also new to this, jumped from web dev tooi just finished the blockchain basics, about to get into the coding aspect. wheww....

**Reply by EngrPips** [reply]
Welcome aboard, Friend.

**Reply by finestxxx** [reply]
thanks a lot, i appreciate friend..i am excited
